Formula/hashlink.rb

```diff
@@ -0,0 +1,68 @@
+class Hashlink < Formula
+  desc "Virtual machine for Haxe"
+  homepage "https://hashlink.haxe.org/"
+  url "https://github.com/HaxeFoundation/hashlink/archive/1.11.tar.gz"
+  sha256 "b087ded7b93c7077f5b093b999f279a37aa1e31df829d882fa965389b5ad1aea"
+  head "https://github.com/HaxeFoundation/hashlink.git"
+
+  depends_on "haxe" => :test
+  depends_on "jpeg-turbo"
+  depends_on "libogg"
+  depends_on "libpng"
+  depends_on "libuv"
+  depends_on "libvorbis"
+  depends_on "mbedtls"
+  depends_on "openal-soft"
+  depends_on "sdl2"
+
+  def install
+    unless build.head?
+      inreplace "Makefile", /\$\{LFLAGS\}/, "${LFLAGS} ${EXTRA_LFLAGS}"
+    end
```

Comment:
Unfortunately, Rubocop and I seem to have one of those love & hate relationships 🤦 
```suggestion
    inreplace "Makefile", /\$\{LFLAGS\}/, "${LFLAGS} ${EXTRA_LFLAGS}" unless build.head?
```
This fixes audit error...